### ISSUE TITLE
Data processing: MASTER Cohort

### ISSUE THREAD

**Issue by famosab** [ghga-de/homogeneous-data-processing#1]
The data from the MASTER cohort can only be processed at DKFZ.

- [ ] process with nf-core/sarek
- [ ] process with DKFZ OTP or Nextflow implementations

---

We start with SNVs and Indels.

**Comment by kubranarci** [comment]
which samples?

**Comment by famosab** [comment]
All samples related to Cholangiocarcinoma!

**Comment by kubranarci** [comment]
Dear @famosab, I read through the documentation https://github.com/ghga-de/homogeneous-data-processing/blob/main/project_overview_ghga_flexfunds-1.pdf and It is spesified there not ALL MASTER COLO samples are open for people outside of HD. So I need more clarification on exactly which samples should be run in the cluster. I think otherwise the first plan was you being the one should running those samples  on our cluster. Please be more specific.

**Comment by famosab** [comment]
Yes I know that not all of them are available. At the moment I cannot access any of them, I opened a Data Access Request back in October via GHGA. I think there is two possibilities: Either we get access for me (which does not work at the moment because my login stopped working) or you try and process the data (at least most of the data that you have access to). I do not know any IDs or similar stuff that I could tell you for the processing (unfortunately).

**Comment by kubranarci** [comment]
Then, I would advise you to get the access (maybe re-request the access) and perform the necessary runs in the cluster yourself. Unfortunately, this project is not my prior responsibility and arrangements would require too much effort from my side for now.
